app/models/miq_event_definition.rb

```diff
@@ -64,7 +64,7 @@ def self.import_from_hash(event, options = {})
 
   def etype
     set = memberof.first
-    raise "unexpected error, no type found for event #{name}" if set.nil?
+    _log.error("No type found for event #{name}") if set.nil?
     set
```

Comment:
Please review the two references to `etype` in https://github.com/ManageIQ/manageiq/blob/master/app/controllers/miq_policy_controller/alerts.rb.